src/DiffSharp.Backends.Torch/Torch.RawTensor.fs

```diff
@@ -124,7 +123,6 @@ type TorchRawTensor(tt: TorchTensor, shape: Shape, dtype: Dtype, device: Device)
 
     override t.GetSlice(fullBounds:int[,]) =
         let n = fullBounds.GetLength(0)
```

Comment:
We can update this to use tensor slicing now https://github.com/xamarin/TorchSharp/blob/master/src/TorchSharp/Tensor/Tensor.cs#L1065

For setting slices: https://github.com/xamarin/TorchSharp/blob/master/src/TorchSharp/Tensor/Tensor.cs#L1095

Reply:
Cool, thanks for reminding. We can do this now and close #277